go/teams/service_helper.go

```diff
@@ -264,16 +264,38 @@ func ReAddMemberAfterReset(ctx context.Context, g *libkb.GlobalContext, teamID k
 	if err != nil {
 		return err
 	}
-	uv := NewUserVersion(upak.Current.Uid, upak.Current.EldestSeqno)
+	uv := upak.Current.ToUserVersion()
 	return RetryOnSigOldSeqnoError(ctx, g, func(ctx context.Context, _ int) error {
 		t, err := GetForTeamManagementByTeamID(ctx, g, teamID, true)
 		if err != nil {
 			return err
 		}
+
+		var existingRole keybase1.TeamRole
+		var existingInvite keybase1.TeamInvite
+		var isAnInvite bool
 		existingUV, err := t.UserVersionByUID(ctx, uv.Uid)
 		if err != nil {
-			return libkb.NotFoundError{Msg: fmt.Sprintf("uid %s has never been a member of this team.",
-				username)}
+			// UV not found - look for invites.
+			invite, uv, found := t.FindActiveKeybaseInvite(uv.Uid)
+			if !found {
+				// username is neither crypto UV nor keybase invite in
+				// that team. bail out.
+				return libkb.NotFoundError{Msg: fmt.Sprintf("uid %s has never been a member of this team.",
+					username)}
```

Comment:
"has never been" -> "is not"
They could have been a member and removed, right? Or is this thing only for implicit teams?

Reply:
I don't think you can remove or add anyone new to implicit team. That's the main reason why we are having these bugs, that the sigchain is supposed to have consistent memberset, and reset people are ruining it.

Reply:
Oh but I guess the question is whether `readd` can be used with normal teams... that's a good question